diff --git a/node_modules/openapi-to-graphql/lib/resolver_builder.js b/node_modules/openapi-to-graphql/lib/resolver_builder.js
index ace3d4d..83232ca 100644
--- a/node_modules/openapi-to-graphql/lib/resolver_builder.js
+++ b/node_modules/openapi-to-graphql/lib/resolver_builder.js
@@ -380,7 +380,7 @@ function getResolver({ operation, argsFromLink = {}, payloadName, data, baseUrl,
             }
             else {
                 // Payload is not an object
-                rawPayload = args[sanePayloadName];
+                rawPayload = args[sanePayloadName] || args[payloadName];
             }
             options.body = rawPayload;
             resolveData.usedPayload = rawPayload;
